src/main/kotlin/build/buf/gradle/BreakingTask.kt

```diff
@@ -14,18 +14,22 @@
 
 package build.buf.gradle
 
+import io.github.g00fy2.versioncompare.Version
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.TaskAction
 
+
 abstract class BreakingTask : DefaultTask() {
     @TaskAction
     fun bufBreaking() {
-        execBuf(
-            "breaking",
-            bufBuildPublicationFile,
-            "--against",
-            singleFileFromConfiguration(BUF_BREAKING_CONFIGURATION_NAME),
-        ) {
+        val args = ArrayList<Any>()
```

Comment:
Kotlin nit: conventionally one should use `mutableListOf<Any>()` unless you specifically need an ArrayList.